shared/styles/base.scss

```diff
@@ -56,12 +62,12 @@ html {
 }
 
 body {
+  @include responsive-font(strip-unit($font-size-min), strip-unit($font-size));
```

Comment:
Feels weird having to strip-unit values here. If the responsive-font mixin needs values to be stripped of units, should the mixin handle that?
Also these values are the default values for the mixin. No need to pass them in.

Reply:
was literally saying the same thing re. strip-unit to @mdmagnusson earlier. will tidy.